sound/soc/sof/intel/cnl.c

```diff
@@ -39,10 +39,6 @@ static irqreturn_t cnl_ipc_irq_thread(int irq, void *context)
 	u32 msg_ext;
```

Comment:
Commit message:
fix missing handling to IPC IRQ -> simplify handling of IPC IRQ